frontend/controller/namespace.js

```diff
@@ -18,16 +18,24 @@ sbp('sbp/selectors/register', {
       return result
     })
   },
-  'namespace/lookup': (name: string) => {
-    // TODO: should `name` be encodeURI'd?
+  'namespace/lookupCached': (name: string) => {
```

Comment:
Part of the idea behind selectors is that they can be expected to be called from multiple different places potentially.

Is there any possibility that this selector will be called by something other than `'namespace/lookup'`? If not, consider either integrating this code back directly into `'namespace/lookup'`, or turning it into a private function in this file that gets called by `'namespace/lookup'`

Reply:
Well, the difference is that the new version only checks the local cache, which is synchronous. This can be useful when you need a response immediately (for example, in `state.js`, although there I'm getting the value directly)